src/lib/Scenes/Artwork/Components/__tests__/ArtworkDetails-tests.tsx

```diff
@@ -58,4 +59,89 @@ describe("Artwork Details", () => {
     expect(component.text()).not.toContain("Frame")
     expect(component.text()).not.toContain("Signature")
   })
+
+  it("shows condition description if present and not biddable", () => {
+    const artworkDetailsInfo = {
```

Comment:
minor: but is it possible to provide a type to this fixture? I think we would want to type with `RequestConditionReport_artwork`

Reply:
good call!

I think the preferred way to do this these days is to use the `@raw_response_type` annotation [(e.g.)](https://github.com/artsy/emission/blob/36878c33dfe49933fb02436a9d11ddc2641c558c/src/lib/Scenes/Collection/Components/__tests__/FeaturedArtists-tests.tsx#L111) on the test query and then index into the generated `*RawResponse` type [(e.g.)](https://github.com/artsy/emission/blob/36878c33dfe49933fb02436a9d11ddc2641c558c/src/lib/Scenes/Collection/Components/__tests__/FeaturedArtists-tests.tsx#L13).

Reply:
👍 I added a type for the fixtures, here the type is `ArtworkDetails_artwork` and there isn't a test query (I didn't write this original test). However I am happy to refactor to use a test query if you want just let me know!